### PR TITLE
Updates for Centos and publishing to Ansible Galaxy.

### PR DESCRIPTION
**Pull request by jefferyfry**

### CODE REVIEW

File: Ansible/README.md [hunk 1]
```diff
@@ -3,33 +3,78 @@
...
+ * [examples directory](examples) - This directory contains example playbooks for various architectures from single Artifactory (RT) deployments to high-availability setups.
  * [infra directory](infra) - This directory contains example infrastructure templates that can be used for testing and as example deployments.
- * [project directory](project) - This directory contains example playbooks for various architectures from single Artifactory (RT) deployments to high-availability setups.
  * [test directory](test) - This directory contains Gradle tests that can be used to verify a deployment. It also has Ansible playbooks for creating infrastructure, provisioning software and testing with Gradle.
  
  ## Getting Started
  
- 1. Install this collection or the roles in your Ansible path using your ansible.cfg file. The following is an example:
+ 1. Download and nstall this collection or the roles in your Ansible path using your ansible.cfg file. The following is an example:
```
Comment (vinayagg) [inline Ansible/README.md:12]: "Install"

File: Ansible/collection/jfrog/ansible/galaxy.yml [hunk 2]
```diff
@@ -3,40 +3,43 @@
...
 
 # A short summary description of the collection
-description: your collection description
+description: "This collection provides roles for installing Artifactory and Xray. Additionally, it provides optional SSL and Postgresql roles if these are needed for your deployment."
 
 # Either a single license or a list of licenses for content inside of a collection. Ansible Galaxy currently only
 # accepts L(SPDX,https://spdx.org/licenses/) licenses. This key is mutually exclusive with 'license_file'
 license:
-- GPL-2.0-or-later
+  - "GPL-2.0-or-later"
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/galaxy.yml:31]: Have we checked with Mark / Legal about which license to use?

File: Ansible/collection/jfrog/ansible/galaxy.yml [hunk 3]
```diff
@@ -45,13 +48,13 @@ tags: []
 dependencies: {}
 
 # The URL of the originating SCM repository
-repository: http://example.com/repository
+repository: "https://github.com/jfrog/JFrog-Cloud-Installers/"
 
 # The URL to any online docs
-documentation: http://docs.example.com
+documentation: "https://www.jfrog.com/confluence/display/JFROG/"
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/galaxy.yml:54]: I think this should be URL to ansible doc

File: Ansible/collection/jfrog/ansible/galaxy.yml [hunk 4]
```diff
@@ -45,13 +48,13 @@ tags: []
...
 
 # The URL to the homepage of the collection/project
-homepage: http://example.com
+homepage: "https://github.com/jfrog/JFrog-Cloud-Installers/"
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/galaxy.yml:57]: I believe this should point to https://github.com/jfrog/JFrog-Cloud-Installers/tree/master/Ansible

File: Ansible/collection/jfrog/ansible/roles/artifactory/meta/main.yml [hunk 5]
```diff
@@ -1,35 +1,16 @@
...
-        - all
-    - name: OpenSUSE
-      versions:
-        - all
-    - name: Ubuntu
-      versions:
-        - bionic
+  issue_tracker_url: "https://github.com/jfrog/JFrog-Cloud-Installers/issues"
+
+  license: license (GPL-2.0-or-later)
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/artifactory/meta/main.yml:8]: which license?

File: Ansible/collection/jfrog/ansible/roles/artifactory_nginx/README.md [hunk 6]
```diff
@@ -0,0 +1,5 @@
+# artifactory_nginx
+This role installs NGINX for artifactory. This role is automatically called by the artifactory role and isn't intended to be used separately.
+
+## Role Variables
+* _server_name_: This is the server name. eg. "artifactory.54.175.51.178.xip.io"
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/artifactory_nginx/README.md:5]: server name sounds like hostname. is this server name or FQDN? 
it says server name, but example is of fqdn

File: Ansible/collection/jfrog/ansible/roles/artifactory_nginx/meta/main.yml [hunk 7]
```diff
@@ -0,0 +1,16 @@
+galaxy_info:
+  author: "Jeff Fry <jefff@jfrog.com>"
+  description: "This role installs NGINX for artifactory. This role is automatically called by the artifactory role and isn't intended to be used separately."
+  company: JFrog
+
+  issue_tracker_url: "https://github.com/jfrog/JFrog-Cloud-Installers/issues"
+
+  license: license (GPL-2.0-or-later)
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/artifactory_nginx/meta/main.yml:8]: which license?

File: Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/README.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
+# artifactory_nginx_ssl
+The artifactory_nginx_ssl role installs and configures nginx for SSL.
+
+## Role Variables
+* _server_name_: This is the server name. eg. "artifactory.54.175.51.178.xip.io"
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/README.md:5]: server name or FQDN?

File: Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/meta/main.yml [hunk 9]
```diff
@@ -0,0 +1,16 @@
+galaxy_info:
+  author: "Jeff Fry <jefff@jfrog.com>"
+  description: "The artifactory_nginx_ssl role installs and configures nginx for SSL."
+  company: JFrog
+
+  issue_tracker_url: "https://github.com/jfrog/JFrog-Cloud-Installers/issues"
+
+  license: license (GPL-2.0-or-later)
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/meta/main.yml:8]: which license.

File: Ansible/collection/jfrog/ansible/roles/postgres/meta/main.yml [hunk 10]
```diff
@@ -1,25 +1,16 @@
...
-      versions:
-        - xenial
-        - bionic
-    - name: Debian
-      versions:
-        - stretch
+
+  issue_tracker_url: "https://github.com/jfrog/JFrog-Cloud-Installers/issues"
+
+  license: license (GPL-2.0-or-later)
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/postgres/meta/main.yml:8]: which license?

File: Ansible/collection/jfrog/ansible/roles/xray/meta/main.yml [hunk 11]
```diff
@@ -1,53 +1,16 @@
...
 
-  # Choose a valid license ID from https://spdx.org - some suggested licenses:
-  # - BSD-3-Clause (default)
-  # - MIT
-  # - GPL-2.0-or-later
-  # - GPL-3.0-only
-  # - Apache-2.0
-  # - CC-BY-4.0
-  license: license (GPL-2.0-or-later, MIT, etc)
+  license: license (GPL-2.0-or-later)
```
Comment (vinayagg) [inline Ansible/collection/jfrog/ansible/roles/xray/meta/main.yml:8]: which license?

File: Ansible/examples/host_vars/xray/hosts.yml [hunk 12]
```diff
@@ -0,0 +1,18 @@
...
+                jfrog_url: http://ec2-18-210-33-94.compute-1.amazonaws.com
+                master_key: "c97b862469de0d94fbb7d48130637a5a"
+                join_key: "9bcca98f375c0728d907cc6ee39d4f02"
+                db_type: "postgresql"
+                db_driver: "org.postgresql.Driver"
+                db_url: "postgres://10.0.0.5:5432/xraydb?sslmode=disable"
+                db_user: "xray"
+                db_password: "xray"
+            hosts:
+                3.17.132.222
```
Comment (vinayagg) [inline Ansible/examples/host_vars/xray/hosts.yml:18]: hard coded ip?

File: Ansible/collection/jfrog/ansible/galaxy.yml (hunk 2 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/galaxy.yml]: Our other public repos (like charts, cli)  are using Apache-2.0 so I changed to that.

https://github.com/jfrog

File: Ansible/collection/jfrog/ansible/galaxy.yml (hunk 3 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/galaxy.yml]: Changed to Ansible collection README.

File: Ansible/collection/jfrog/ansible/galaxy.yml (hunk 4 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/galaxy.yml:58]: Done.

File: Ansible/collection/jfrog/ansible/roles/artifactory/meta/main.yml (hunk 5 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/roles/artifactory/meta/main.yml]: Now Apache-2.0.

File: Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/meta/main.yml (hunk 9 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/roles/artifactory_nginx_ssl/meta/main.yml]: Now Apache-2.0

File: Ansible/collection/jfrog/ansible/roles/postgres/meta/main.yml (hunk 10 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/roles/postgres/meta/main.yml]: Now Apache-2.0.

File: Ansible/collection/jfrog/ansible/roles/xray/meta/main.yml (hunk 11 above)
Comment (jefferyfry) [reply Ansible/collection/jfrog/ansible/roles/xray/meta/main.yml]: Now Apache-2.0.

File: Ansible/examples/host_vars/xray/hosts.yml (hunk 12 above)
Comment (jefferyfry) [reply Ansible/examples/host_vars/xray/hosts.yml:18]: This is fine. It is just an example file.

**Pull request merged**